packages/repository/src/relations/belongs-to/belongs-to.repository.ts

```diff
@@ -17,6 +17,8 @@ export interface BelongsToRepository<Target extends Entity> {
   /**
    * Gets the target model instance
    * @param options
+   * @returns A promise of the target object, or throws an EntityNotFound error
+   * when target model instance not found.
```

Comment:
Does it throw an error or return a Promise that rejects with the error?

Reply:
Returns 
- A promise of the target object, ( if found)
-  or throws an EntityNotFound error when target model instance not found ( not found)

How about :
`returns A promise of the target object, otherwise throws...`?

Reply:
I believe this API is implemented as an `async` function, therefore the `EntitNotFound` error thrown by the code is automatically converted to a rejected promise by JS runtime.

Reply:
How about the following:

> A promise resolved with the target object or rejected with an `EntityNotFoundError`
when target model instance was not found.